geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DescribeRegionDUnitTest.java

```diff
@@ -145,10 +155,9 @@ public void describeLocalRegionOnlyOneServer1() throws Exception {
   @Test
   public void describeRegionWithCompressionCodec() throws Exception {
     final String regionName = "compressedRegion";
-    VM vm = Host.getHost(0).getVM(1);
 
     // Create compressed region
-    vm.invoke(() -> createCompressedRegion(regionName));
+    server1.invoke(() -> createCompressedRegion(regionName));
```

Comment:
I like the refactor here.

However, I don't love that this test is altering the cluster state for all tests.  That `destroyRegion` at the end isn't guaranteed in any way.

I'd prefer it if the compressed region's creation can be added to the `@BeforeClass`, if that wouldn't upset the other tests.  Otherwise, the `@BeforeClass` should become a `@Before` or this test should have its own class.

Reply:
Done.